### ISSUE TITLE
new image handling in dayplans

### ISSUE THREAD

**Issue by urapadmin** [urapadmin/kiosk#487]
### @test
apply the general test for image containers on dayplans

**Comment by urapadmin** [comment]
https://www.meritaten.net/urapdev/doku.php?id=how_to_test_an_image_container

**Comment by SamKimball-BrownU** [comment]
Works as intended except for using backspace to delete dayplan records
When using backspace to delete, it remains searchable both by the description and the identifier.

**Comment by SamKimball-BrownU** [comment]
Error encountered in other testing:

When testing collected image handling (ticket #489 ), I added some photos to a new excavation unit that I called Gallery under dayplan. The first two images I added were completely fine. I used them to test the gallery input for dayplan image handling (in a different unit). Then, when I tried to test collected image handling, I did the same thing. Except this time, they not in the gallery under any identifier. I uploaded another image, and the issue did not repeat.

I was unable to replicate the error. This probably isn't super helpful, but I wanted to document this random bug I encountered.

**Comment by urapadmin** [comment]
I tried and tried and tried, but backspace works ok in my version. Please try again and this time tell me, where exactly did you use backspace? In the popup or in the primary container? When you can still find an image, please look in the additional info (the button in the upper right corner of an image in the gallery). Does it really say dayplan, like here:
![image](https://user-images.githubusercontent.com/38838314/76485838-fbbea700-63f3-11ea-9be1-19cb582d2deb.png)

**Comment by SamKimball-BrownU** [comment]
USTP 1.9.11, Windows 10

Issue with backspace persists
![image](https://user-images.githubusercontent.com/61027113/79053889-60913b00-7c06-11ea-8c0e-23144c96f401.png)

this is the area that is selected when I use backspace, a popup happens and then it deletes
![image](https://user-images.githubusercontent.com/61027113/79053917-8fa7ac80-7c06-11ea-95b7-1802d1451afb.png)

If no further images are added, then it still appears in the preview. See image below.
This is after deleting record using backspace
![image](https://user-images.githubusercontent.com/61027113/79053996-34c28500-7c07-11ea-8273-d95d4d1fcfe3.png)

**Comment by SamKimball-BrownU** [comment]
Oh! additional piece of information: issue does not happen when I use backspace to delete the image itself. It's only in this very specific situation. I would say this is something that an actual user is very very unlikely to ever encounter.

**Comment by urapadmin** [comment]
I make a new ticket for the unrelated backspace / del issue in the dayplan list: #

**Comment by urapadmin** [comment]
### fixed in v8.9.14
In this case even drag & drop should work properly because we have only a single container.

**Comment by SamKimball-BrownU** [comment]
- [ ] Simple
- [ ] not-simple
- [ ] feature
- [ ] survey

**Comment by SamKimball-BrownU** [comment]
Issue with feature: separating out into new ticket, see #615, otherwise it is working as intended

**Comment by SamKimball-BrownU** [comment]
Same thing happens in survey units, see #615 again

**Comment by SamKimball-BrownU** [comment]
Other than that, I could find no issues. This will need testing again after #615 is resolved, most likely, so I am leaving it open

**Comment by SamKimball-BrownU** [comment]
Woo Hoo! everything's resolved now. Thank goodness.

**Comment by urapadmin** [comment]
reopened because image proxy was not set correctly.

**Comment by urapadmin** [comment]
### v9.1
next round.
